### DISCUSSION TITLE
Treeshake output

### DISCUSSION THREAD

**Discussion by adamivancza** [microsoft/rnx-kit#2765] [Q&A]
Would it be possible to see what pieces of code are being removed due to treeshaking? If this is in our codebase it would make sense to remove those ourselves if it is not truly used anywhere.

**Comment by BrentonWheeler** [comment]
You could use something like https://github.com/IjzerenHein/react-native-bundle-visualizer, it also has a JSON format output option which you could try `diff` the before vs after outputs
